packages/experimental/Shimmer/src/Shimmer.types.ts

```diff
@@ -0,0 +1,135 @@
+import { SvgProps, RectProps, CircleProps, ImageProps } from 'react-native-svg';
+import { ImageURISource } from 'react-native';
+
+export const shimmerName = 'Shimmer';
+/**
+ * Specifies the possible type of the shimmer elements.
+ * Required when building more than one element at a time.
+ */
+export enum ShimmerElementType {
+  /**
+   * Rect element type
+   */
+  rect = 1,
+  /**
+   * Circle element type
+   */
+  circle = 2,
+}
```

Comment:
nit: I've seen people do something more like this for enums
```
export ShimmerElementType = 'rect'  | 'circle' 
```

I'm not sure, but maybe that's more the more standard way to do enums now? I'm not sure of the pros/cons but I'd love to learn more about it if others know..

Reply:
Read up on enums in typescript. I learned that numeric enums aren't type-safe and it's unnecessary in our case so it's probably better to use union types.

Reply:
Oh wow! Today I learned that the unions are more type safe. Thanks for looking into it :D